### PR TITLE
LFS: ABI fixes

### PR DESCRIPTION
**Pull request by no92**
This possibly needs to be merged into `abi-break` but that needs updating first.

### CODE REVIEW

File: options/linux/include/sys/epoll.h [hunk 1]
```diff
@@ -40,7 +43,11 @@ typedef union epoll_data {
 struct epoll_event {
 	uint32_t events;
 	epoll_data_t data;
-};
+}
```
Comment (Geertiebear) [inline options/linux/include/sys/epoll.h:46]: Why not just always packed?

Comment (no92) [reply options/linux/include/sys/epoll.h:46]: My host (glibc, via a test program), musl and bionic seem to agree that this is correct and only necessary for x86_64.

**Review by Dennisbonke — APPROVED** [review]
LGTM outside that one comment.

**Review by Dennisbonke — APPROVED** [review]
LGTM

**Pull request merged**